perl.html.markdown

```diff
@@ -126,18 +126,20 @@ while (condition) {
 
 
 # for loops and iteration
-for (my $i = 0; $i < $max; $i++) {
+for my $i (0 .. $max) {
   print "index is $i";
 }
 
-for (my $i = 0; $i < @elements; $i++) {
+while (my $i = each @elements) {
```

Comment:
I don't remember seeing that for an array. Is it really idiomatic? Also might warrant a version using `$#`